app/views/questions/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<h2>Question</h2>
+
+<div class="row">
+  <div class="col s12 m12">
+    <div class="card-panel white">
+      <span class="black-text">
+          <%= form_for @question do |f| %>
+        <div id="question-title" >
```

Comment:
With this being the edit page, it would probably be a good idea to have the values for each field pre-populated with the data that already exists for each field.